src/GafferImage/ColorSpace.cpp

```diff
@@ -102,8 +108,20 @@ void ColorSpace::hashTransform( const Gaffer::Context *context, IECore::MurmurHa
 
 	inputSpacePlug()->hash( h );
 	outputSpacePlug()->hash( h );
+
+	std::string name;
```

Comment:
Again, can this not just be moved `OpenColorIOTransform::hashColorData()`? Note that duplicating it in the derived classes instead has already created a bug - the version here isn't hashing the name, but the version in `DisplayTransform::hashTransform()` is. It's the latter one we need to keep.